zebra/zebra_vxlan_private.h

```diff
@@ -65,6 +65,9 @@ struct zebra_vni_t_
   /* VNI - key */
   vni_t vni;
 
+  /* Flag for advertising gw macip */
+  u_int8_t  advertise_gw_macip;
```

Comment:
`uint8_t` is standard since C99.

I'd like to get rid of all of these at some point